resources/authgear/templates/en/web/authflowv2/settings_profile.html

```diff
@@ -173,9 +173,12 @@
     {{ $url = $.EditURL }}
   {{ end }}
 
-  {{ $content := (translate "v2.page.settings-profile.default.attribute-value-unspecified-label" nil) }}
-  {{ if $.Content }}
-    {{ $content = $.Content }}
+  {{ $content := $.Content }}
+  {{ if (eq $.Content nil) }}
```

Comment:
I remember we use `isNil` to check `nil`

Reply:
f1b0f7edee5bc74666b42a10e0f0e03c228553ed

Reply:
Updated 🙏